tests/main.test.ts

```diff
@@ -61,5 +61,6 @@ describe("Run tests", () => {
     expect(parseDurationString("Time: <1 Week").get("weeks")).toEqual(1);
     expect(parseDurationString("Time: <4 Hours").get("hours")).toEqual(4);
     expect(parseDurationString("Time: <8 Weeks").get("months")).toEqual(2);
+    expect(parseDurationString("Time: <3 Months").get("months")).toEqual(3);
```

Comment:
Is it worth adding tests for non plural as well? i.e. `Time: <1 Month` as well as for the other resolutions i.e. `Hour`, `Day`

Reply:
Added singular / plural for each unit.